public/app/features/alerting/unified/components/rule-editor/util.ts

```diff
@@ -144,10 +144,14 @@ export type ThresholdDefinitions = Record<string, ThresholdDefinition>;
 /**
  * This function will retrieve threshold definitions for the given array of data and expression queries.
  */
-export function getThresholdsForQueries(queries: AlertQuery[]) {
+export function getThresholdsForQueries(queries: AlertQuery[], condition: string | null) {
```

Comment:
maybe we should update this function name to  `getThresholdInCondition` ,as we are not going to return a list of thresholds but only one?.

Reply:
Hm no there can still be more than one threshold (when using classic condition) where multiple conditions are supported.

Reply:
ahhh! ok!!